data/uhc_pack/functions/starting/start.mcfunction

```diff
@@ -77,6 +77,9 @@ execute if score UHCShowAdvs uhcEnabled matches 0 run gamerule announceAdvanceme
 execute if score UHCFriendlyFire uhcEnabled matches 1 run function uhc_pack:starting/friendlyfire_enable
 execute if score UHCFriendlyFire uhcEnabled matches 0 run function uhc_pack:starting/friendlyfire_disable
 
+# Set starting health
+execute run function uhc_pack:starting/starting_health
```

Comment:
This only needs to be `function uhc_pack:starting/starting_health`. `execute` is only required if you are changing the location of execution or making it conditional.